.github/workflows/pkgci_regression_test_amdgpu_rocm.yml

```diff
@@ -17,10 +17,15 @@ on:
         type: string
         default: ""
 
+permissions:
+  contents: read
```

Comment:
I thought the read access was granted by default? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Could you elaborate what the problem is without this change?

Reply:
Oh, I found the original discussion (https://github.com/openxla/iree/pull/16408). Thanks for restricting the permissions.

Reply:
> Oh, I found the original discussion (#16408). Thanks for restricting the permissions.

Yeah, sorry for not providing more context here. For completeness, I cross post here:

At the moment, the default permissions are still "Read/write for all scopes (current default)", see https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/. Therefore, restricting the permissions as proposed among others by the OpenSSF.